test/runner.html

```diff
@@ -8,16 +8,6 @@
 <body>
   <div id="mocha"></div>
   <script src="/node_modules/mocha/mocha.js"></script>
-  <script src="/node_modules/should/should.js"></script>
-  <script src="http://amo.alicdn.com/L1/377/101010/sea.js"></script>
-  <script src="/config.js"></script>
-  <script>
-    /* eslint-disable */
-    mocha.ui('bdd')
-    define('should', should)  // use the pre-build version of should
-    seajs.use('heredoc/test/runner', function() {
-      mocha.run()
-    })
-  </script>
```

Comment:
This PR is about this change mostly. There's no more strange stuff here. Hope you like it @jden